### ISSUE TITLE
Can i precise if a face is unfoldable

### ISSUE THREAD

**Issue by amar1323** [amandaghassaei/OrigamiSimulator#31]
Thank you so much for this wonderful tool. Can i precise somehow that a face is unfodable? i have an svg file where the base of my origami is a polygon that i want to preserve it during the folding process.

Thank you in advance

**Comment by edemaine** [comment]
No, this is not a currently supported option. It's an interesting idea, though I'm not sure how it would be naturally specified &mdash; maybe clicking on a face in the user interface, under Advanced Options?
